snap/snapcraft.yaml

```
@@ -58,10 +58,8 @@ plugs:
   kube-config:
     interface: personal-files
     read:
-    - $HOME/.kube/config
-    - $HOME/.minikube/client.key
-    - $HOME/.minikube/client.crt
-    - $HOME/.minikube/ca.crt
+    - $HOME/.kube
+    - $HOME/.minikube
```

To support `k3d` we would need access to the host's `$XDG_CONFIG_HOME`, but that does not appear to be supported well by `snap`: [thread](https://forum.snapcraft.io/t/accessing-xdg-config-home-of-the-host/9780/10)

We can handle the common case by just hard-coding `~/.config`:
```suggestion
    - $HOME/.minikube
    - $HOME/.config/k3d
```

---

Thanks a lot for digging into this!